soccer/gameplay/plays/restarts/basic_indirect.py

```diff
@@ -0,0 +1,53 @@
+import standard_play
```

Comment:
This file shouldn't appear in the diff after we merge my changes and you pull again.